.github/workflows/spring-boot-build-publish-image-config.yml

```
@@ -17,6 +17,11 @@ on:
         default: 'latest'
         required: false
         type: string
+      container-scan-enabled:
+        description: Container-scan is default enabled, disable this with caution since vulnerabilities might enter production without scan
+        default: true
```

Foretrekker at den skrus av _per default_
```suggestion
        default: false
```